### DISCUSSION TITLE
升级2.11.0连不上fcm了

### DISCUSSION THREAD

**Discussion by misakaok** [MetaCubeX/ClashMetaForAndroid#337] [General]
分应用代理后不行了

![Screenshot_2024-09-13-17-39-41-648_com.google.android.gms.jpg](https://github.com/user-attachments/assets/e97105b3-02c0-460b-87c3-96f13fd8b266)

**Comment by ericpq** [comment]
1.FCM服务器*mtalk.google.com,并没被墙，可直连。
2.光靠rules进行分流好像不太行，建议配合hosts
3.MIUI或者澎湃，因为电池管理原因，锁屏后FCM会断开连接，需要用国际版电池性能进行替换。
![Screenshot_2024-09-14-21-35-39-237_com.google.android.gms.jpg](https://github.com/user-attachments/assets/83c71112-316d-4e6e-8d58-22b5996ee2d0)

**Comment by entr0pia** [comment]
加个hosts https://github.com/entr0pia/fcm-hosts

**Reply by ghzxs** [reply]
怎么加到配置文件里
